components/table/test/table.vdiff.js

```diff
@@ -664,6 +664,26 @@ describe('table', () => {
 					await expect(elem).to.be.golden();
 				});
 
+				it('col-sort-button-dropdown-open-short-flag-off', async() => {
+					window.D2L = {};
+					const elem = await createTableFixture(html`
+						<thead>${createSortableButtonDropdownHeaderRow()}</thead>
+						<tbody>${createRows([1])}</tbody>
+					`);
+					await clickElem(elem.shadowRoot.querySelector('d2l-table-col-sort-button'));
+					await expect(elem).to.be.golden();
+				});
```

Comment:
Not totally necessary, though it shows the problem. Would eventually get removed.